LogicalTypes.md

```diff
@@ -366,3 +366,7 @@ optional group my_map (MAP_KEY_VALUE) {
 }
 ```
 
+## Null
+Sometimes when discovering the schema of existing data values are always null and there's no type information.
```

Comment:
I think the merge logic stays the same in distributed mode as long as the full type is carried along and we know it's null.